nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/pom.xml

```diff
@@ -124,6 +124,21 @@
             <artifactId>guava</artifactId>
             <version>27.0.1-jre</version>
         </dependency>
+        <dependency>
+            <groupId>com.fluenda</groupId>
+            <artifactId>parcefone</artifactId>
+            <version>2.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.bval</groupId>
+            <artifactId>bval-jsr</artifactId>
+            <version>1.1.2</version>
```

Comment:
This particular version is several years old, the current version is 2.0.5, is it possible to use that version?

Reply:
You are right in both cases: I used versions from the original CEF processor implementation. Note: BVAL needed validation dependency as well, as new BVAL needs Java Validation API 2.0